software-update/fpga_list.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+ls /etc/fpga/
+mount /dev/sda1 /mnt
+mkdir /temp_fpga
+cp /mnt/*.jam /temp_fpga
+umount /mnt
```

Comment:
Fpga: /etc/fpga
Bios: /etc/bios
FC: /etc/px4-fw and /etc/ardupilot-fw

If the user wants to flash a custom one, we should give an option for him to choose the file on his computer  and upload. The plugin then saves this to a temporary location and proceed with flashing.

Makes sense?